containers/python-3-pypy/README.md

```diff
@@ -0,0 +1,52 @@
+# PyPy (Community)
+
+## Summary
+
+*Develop python applications using the [PyPy](https://www.pypy.org/) interpreter*
+
+| Metadata                    | Value                                                                        |
+|---------------------------- | -----------------------------------------------------------------------------|
+| *Contributors*              | [Cyb3r Jak3](https://github.com/Cyb3r-Jak3)                                  |
+| *Categories*                | Community, Languages                                                         |
+| *Definition type*           | Dockerfile                                                                   |
+| *Works in Codespaces*       | Yes                                                                          |
+| *Container host OS support* | Linux, macOS, Windows                                                        |
+| *Container OS*              | Debian                                                                       |
+| *Languages, platforms*      | [Languages and platforms the container supports]                             |
```

Comment:
You can add the languages here since this goes into a suggestion algorithm.

```suggestion
| *Languages, platforms*      | Python, PyPy                            |
```